docs/contributing/dependencies/index.md

```diff
@@ -27,6 +27,8 @@ The default configuration:
   the workload.
 - Manages updates with semantic versioning and lock file updates. Rebases are disabled.
 - Allows an unlimited number of pull requests to be created.
+- Delays creation of pull requests until at least seven days after the dependency version release,
+  to ensure any post-release security issues are addressed.
```

Comment:
```suggestion
  to ensure stability of said release from bugs and / or security issues.
```

🎨 I think we should be a bit less direct that this is about security, because we see it as a general stability measure. Feel free to revise further.